pages/common/rustc.md

```diff
@@ -1,33 +1,33 @@
 # rustc
 
 > The Rust compiler.
-> Processes, compiles and links Rust language source files.
+> Rust projects usually use `cargo` instead of invoking `rustc` directly.
 > More information: <https://doc.rust-lang.org/rustc>.
 
-- Compile a single file:
+- Compile a binary crate:
```

Comment:
I am not sure if a single file can be called a `crate`.
IMO, cargo uses `rustc` internally to compile different files as a crate.

Reply:
Today I learnt. Ignore my comment :)